Jenkinsfile

```diff
@@ -0,0 +1,50 @@
+#!groovy
+
+def newImage = null
+
+pipeline {
+  agent none
+
+  options {
+    quietPeriod(120) // builds happen at least 120 seconds apart
+    disableConcurrentBuilds()
+  }
+
+  stages {
+
+    stage('Build Docker image and stage site') {
+      agent any
+
+      steps {
+        script {
+          def dockerRepoName = 'zooniverse/translations'
+          def dockerImageName = "${dockerRepoName}:${GIT_COMMIT}"
+          newImage = docker.build(dockerImageName)
+          newImage.inside {
+            sh """
+              cd /src
+              npm run --silent stage
```

Comment:
not blocking. i can see branch deploy clobbering each other especially with dependabot branches... if this is useful then great but i'm not sure it is.

Reply:
I've been manually deploying PR branches to https://pandora.zooniverse.org so I figured this is one less step to remember. I should open another PR to add the deployed commit so that we can see what's currently deployed.